docs/learn/bitesize/bitesize-socialhub.md

```diff
@@ -1,30 +1,32 @@
 ---
-id: bitesize-usecases
-title: Multiplayer Use Cases sample
+id: bitesize-socialhub
+title: Multiplayer Distributed Authority Social Hub sample
 description: Learn about the scenes in the Multiplayer Use Cases sample.
 ---
 
-The [Distributed Authority Social Hub Sample](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize/tree/main/Basic/DistributedAuthoritySocialHub) is a project aimed to demonstrate Distributed Authority's core features and benefits, helping users seamlessly integrate Distributed Authority into their own game projects.
-Within Social Hub, you are free to explore Distributed Authority's key features & functionalities in a sandbox-like environment. Such features include:
+The [Distributed Authority Social Hub Sample](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize/tree/main/Basic/DistributedAuthoritySocialHub) is a project that demonstrates Distributed Authority's features and helps you integrate Distributed Authority into your own game projects.
 
-* Automatic redistribution of ownership of NetworkObjects across connected clients
-* Client-side NetworkObject spawning and deferred NetworkObject despawning
-* Ownership transfer of NetworkObjects with variable ownership flags
-* Host migration of a networked game session
+Within Social Hub, you can explore Distributed Authority in a sandbox-like environment. You can:
+
+- Automatically redistribute ownership of NetworkObjects across connected clients
+- Client-side NetworkObject spawning and deferred NetworkObject despawning
+- Transfer ownership of NetworkObjects with variable ownership flags
+- Host migration of a networked game session
 
 ## The Bootstrap scene
 
-This is the entry point to the sample. Once loaded, you will be transitioned to the MainMenu scene.
+This is the entry point to the sample. After it loads, the MainMenu scene appears.
```

Comment:
Does this sentence mean after the Bootstrap scene loads, we see the MainMenu scene? Or after the MainMenuScene loads, we see the MainMenu scene? 

What's the point of the Bootstrap scene? Is this the Welcome window with the pigs?

Reply:
Yes, the former.
Bootstrap scene contains the NetworkManager instance that will, for now, be the single NetworkManager that will persist throughout mutliple game sessions.
This Bootstrap scene also is the only place in code where we initialize the player with Unity Services. If the player has failed to register the project with a cloud id, the main menu scene won't load.

But that's a good callout, in our game flow, and in this section - we need to alert the player to add a cloud id.

In this doc, though, we can grab this bit from our readme:

> Social Hub leverages several services from UGS to facilitate connectivity between players. To use these services inside your project, you must [create an organization](https://support.unity.com/hc/en-us/articles/208592876-How-do-I-create-a-new-Organization-) inside the Unity Dashboard. Consult the Multiplayer SDK [Getting Started](https://docs.unity.com/ugs/en-us/manual/mps-sdk/manual/get-started) manual for more resources.